### ISSUE TITLE
gradle 6.3 support

### ISSUE THREAD

**Issue by TheForgetTime** [openjfx/javafx-gradle-plugin#76]

**Comment by msgilligan** [comment]
I use this plugin with Gradle 6.3 and it works fine. What is missing or not working properly?

**Comment by TheForgetTime** [comment]
Maybe it is related to my use of kotlin in the modular java project

**Comment by TheForgetTime** [comment]
So I have multiple source program compilation paths
its looks like
It looks like this:
```
build
-java
    -com.xxx
    -module-info.class
-kotlin
    -com.xxx
```

**Comment by belu** [comment]
There is a known problem with Gradle 6.3 and jlink. Do you get an error like this:
```
Execution failed for task ':compileJava'.
> The value for task ':compileJava' property 'destinationDirectory' is final and cannot be changed any further.
```

For more details, see this issue:
https://github.com/beryx/badass-jlink-plugin/issues/116

When I switched back to Gradle 6.0.1 it worked for me. However, then you are not able to use JDK 14 anymore as this is not supported before version 6.3.

**Comment by xipgroc** [comment]
gradle-modules-plugin just released 1.7.0, upgrading should add support for modules + gradle 6.4 right? 
https://github.com/java9-modularity/gradle-modules-plugin/releases/tag/v1.7.0

**Comment by abhinayagarwal** [comment]
This has been fixed and one can use v0.0.9
